lms/templates/courseware/progress.html

```diff
@@ -57,8 +57,9 @@ <h1 class="progress-certificates-title">${_("Course Progress for Student '{usern
 
       <section class="progress-info-mouseover">
         <p>
-            Note: Submitted assignments will appear as bars in the Progress chart.
-            Mouse over a bar to see more information about the relevant assignment.
+            ${_('''Note: Submitted assignments will appear as bars in the Progress chart.
```

Comment:
It may be _used_ as a "translation string", but since it uses 3 consecutive quotation marks, I'd still call it a docstring.

I think that use is more incidental than intentional:

Triple quotes are recommend by PEP257 [1] and edX [2].

[1] https://www.python.org/dev/peps/pep-0257/#id15
[2] https://www.python.org/dev/peps/pep-0257/#id15